README.md

```diff
@@ -147,6 +145,8 @@ const category = 'element';
 oTracking.click.init(category);
 ```
 
+_Note: The attribute `data-o-tracking-skip-queue` is no longer used, it is now the default behaviour._
```

Comment:
this readme error is because it's inconsistent with line 35's recommendation

Reply:
Ahh so it's about consistency. I was mildly annoyed, I prefer the underscore syntax :D Not enough to update the readme though, I'll switch this line

Reply:
i prefer underscore for emphasis and double star for bold